<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HuffmanCoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="mStatus">
        <property name="text">
         <string>Select a file to compress/decompress</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="mSelectedFile">
          <property name="placeholderText">
           <string>Enter a .txt file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mBrowseFiles">
          <property name="text">
           <string>Browse files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="mCompress">
          <property name="text">
           <string>Compress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mDecompress">
          <property name="text">
           <string>Decompress</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
